tasks/build.yml

```diff
@@ -59,14 +59,78 @@
       become: "no"
       when:
         - not atom_compile_all_themes|bool # Don't compile themes when they are alreay compiled
-  when: ansible_os_family == "RedHat"
+  when:
+    - ansible_os_family == "RedHat"
+    - atom_php_version|int < 74
+
+- name: "Build AtoM themes (CentOS/RH and php version >= 7.4)"
+  block:
+    - name: "Build all AtoM themes (CentOS/RH)"
+      shell: "make"
+      args:
+        chdir: "{{ item }}"
+      with_items: "{{ theme_makefiles.stdout_lines }}"
+      become: "no"
+      when:
+        - atom_compile_all_themes|bool
+    - name: "Build selected AtoM themes (CentOS/RH)"
+      shell: "{{ item.build_cmd }}"
+      args:
+        chdir: "{{ item.path }}"
+      with_items: "{{ atom_themes }}"
+      become: "no"
+      when:
+        - not atom_compile_all_themes|bool # Don't compile themes when they are alreay compiled
+  when:
+    - ansible_os_family == "RedHat"
+    - atom_php_version|int >= 74
+
+#
+# Bootstrap 5 support
+#
+
+- name: "Prepare npm to be used with bootstrap5 themes"
+  block:
+    - name: "Update npm to latest version of its release"
+      command: "npm install -g npm@latest"
+    - name: "Get AtoM user home directory"
+      become_user: "{{ atom_user }}"
+      become: "yes"
+      shell: >
+             getent passwd {{ atom_user }}  | awk -F: '{ print $6 }'
+      changed_when: false
+      register: __atom_user_home_directory
+    - name: "Create npm directories on {{ atom_user }} home dir"
+      become: "yes"
+      file:
+        path: "{{ __atom_user_home_directory.stdout }}/{{ item }}"
+        state: "directory"
+        owner: "{{ atom_user }}"
+        group: "{{ atom_group }}"
+      loop:
+        - ".config"
+        - ".npm"
+    - name: "Clean cache, install and build with npm on AtoM site directory"
+      become: "yes"
+      become_user: "{{ atom_user }}"
+      command: "{{ item }}"
+      args:
+        chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+      environment:
+        CYPRESS_INSTALL_BINARY: 0
+      loop:
+        - "npm cache clean --force"
+        - "npm install"
+        - "npm run build"
+  when:
+    - atom_php_version|int >= 74
```

Comment:
(more a question than a change request, not sure about ) Is BS5 support required for AtoM 2.7? or when php > 7.4 or (php > 74) ? or both conditions are needed ?

Reply:
Feedback from @mamedin :

> BS5 support is optional on AtoM 2.7, but when installing AtoM 2.7 with BS5 support, the BS2 themes are still working. The strategy we followed when upgrading the customers that wanted to test AtoM 2.7 was: Always add BS5 support.
> 
> To reduce the conditionals in role, I though we can follow the same strategy for AtoM 2.7, but... I decided to use the atom_php_version|int >= 74 conditional because this is the easiest way to know that it is an AtoM 2.7 (or greater). There is no other variable directly related to AtoM version (except atom_es_config_version).
> 
> Other option is to create a new variable, like atom_bootstrap5_support.

> Checking the atom_php_version variable in the master branch, it is useless for Ubuntu20, never used. By mistake I was testing with ubuntu20 and atom_php_version: 72, and all worked fine because the atom_php_version var was only used to install the bootstrap support.
> 
> The desired variable for the BS5 support conditional should be php_version (see vars/DISTRO.yml files) instead of atom_php_version. To remove the dot, we can use php_version|replace('.','')|int >= 74
>